skl2onnx/operator_converters/one_vs_one_classifier.py

```diff
@@ -0,0 +1,175 @@
+from .._supported_operators import sklearn_operator_name_map
+
+
+def convert_one_vs_one_classifier(scope: Scope, operator: Operator,
+                                   container: ModelComponentContainer):
+
+    proto_dtype = guess_proto_type(operator.inputs[0].type)
+    if proto_dtype != onnx_proto.TensorProto.DOUBLE:
+        proto_dtype = onnx_proto.TensorProto.FLOAT
+    op = operator.raw_operator
+    options = container.get_options(op, dict(raw_scores=False))
+    use_raw_scores = options['raw_scores']
+    label_names = []
+    score_names = []
+    for i, estimator in enumerate(op.estimators_):
+        op_type = sklearn_operator_name_map[type(estimator)]
+        label_name_out = 'label_out_%d' % i
+        score_name_out = 'score_out_%d' % i
+        if op_type == 'SklearnLinearSVC':
+            attrs = {
+                'coefficients': estimator.coef_.ravel().tolist(),
+                'intercepts': estimator.intercept_.ravel().tolist()
+            }
+            container.add_node(
+                op_type = 'LinearClassifier', 
+                inputs = operator.inputs[0].full_name,
+                outputs = [label_name_out, score_name_out],
+                name = 'LinearClassifier_%d' % i, 
+                op_domain='ai.onnx.ml', **attrs
+            )
+
+        label_names.append(label_name_out)
+        score_names.append(score_name_out)
+    # end for
+
+    # label
+    apply_concat(scope, label_names, 'concat_label_out', container, axis=0)
+
+    container.add_initializer('const_1', onnx_proto.TensorProto.FLOAT, [], [1])
+    container.add_initializer('const_3', onnx_proto.TensorProto.FLOAT, [], [3])
+    container.add_initializer(
+        'target_shape', onnx_proto.TensorProto.INT64, 
+        [2], [op.n_classes_, -1])
+
+    apply_reshape(
+        scope, 'concat_label_out', 'reshape_label_out',
+        container, desired_shape='target_shape')
+
+    apply_transpose(
+        scope, 'reshape_label_out', 'transpose_label_out', 
+        container, perm=(1, 0))
+
+    apply_cast(
+        scope, 'transpose_label_out', 'cast_label_out', 
+        container, to=onnx_proto.TensorProto.FLOAT
+    )
+
+    # score
+    apply_concat(
+        scope, score_names, 'concat_score_out',
+        container, 'concat_score', axis=0)
+
+    apply_reshape(
+        scope, 'concat_score_out', 'reshape_score_out',
+        container, desired_shape='target_shape')
+
+    apply_transpose(
+        scope, 'reshape_score_out', 'transpose_score_out',
+        container, perm=(1, 0))
+
+    n_classes = len(op.classes_)
+    sumc_name = [scope.get_unique_variable_name('svcsumc_%d' % i)
+                 for i in range(n_classes)]
+    vote_name = [scope.get_unique_variable_name('svcvote_%d' % i)
+                 for i in range(n_classes)]
+    sumc_add = {n: [] for n in sumc_name}
+    vote_add = {n: [] for n in vote_name}
+    k = 0
+    for i in range(n_classes):
```

Comment:
You should use this operator: https://github.com/onnx/sklearn-onnx/blob/main/skl2onnx/operator_converters/ovr_decision_function.py#L154. The code is the same.